### DISCUSSION TITLE
Congratulations on the 1.0 release!

### DISCUSSION THREAD

**Discussion by msfjarvis** [pgpainless/pgpainless#230] [General]
Hey Paul,

I'm one of the maintainers for [Android Password Store](https://github.com/android-password-store/Android-Password-Store) which uses GPG for encryption. We're using PGPainless to implement a new, in-app [GPG backend](https://github.com/android-password-store/Android-Password-Store/tree/develop/crypto-pgpainless) to replace the deprecated [open-keychain](https://github.com/open-keychain/open-keychain) and have been thoroughly pleased and impressed by the quality of your work here. Thanks for your efforts in making GPG suck significantly lesser!

Cheers,
Harsh

**Comment by Valodim** [comment]
Good to see. OpenKeychain was always a questionable match for Android Password Store :+1:

**Comment by vanitasvitae** [comment]
Hey Harsh,

Thank you for the fish :D
Awesome to see how the word spreads even though there hasn't been an official announcement yet.

**Reply by msfjarvis** [reply]
We've been using PGPainless since at least the `0.2.17` release and I've personally kept a close eye, so I was particularly excited once the 1.0.0 RCs began. I might reach out with questions in the near future as we expand on the new GPG backend, hope that's alright :)

**Reply by wiktor-k** [reply]
@msfjarvis Do you plan to use the [OpenKeychain-as-a-library SDK](https://github.com/cotechde/hwsecurity) from OKC authors?  I'm wondering if more advanced use cases such as using tokens to decrypt data are still in scope for Password Store :)

**Reply by msfjarvis** [reply]
We do, but not for GPG. The relevant discussion for it can be found [here](https://github.com/android-password-store/Android-Password-Store/issues/1486).

**Reply by wiktor-k** [reply]
Wow, great to hear and see such a roster of improvements being done! Didn't know age will be also supported! 🤯️
